modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -256,6 +268,10 @@ public CompletableFuture<Void> alterZone(String name, DistributionZoneConfigurat
             return failedFuture(new IllegalArgumentException("Distribution zone configuration is null"));
         }
 
+        if (DEFAULT_ZONE_NAME.equals(name) && !DEFAULT_ZONE_NAME.equals(distributionZoneCfg.name())) {
+            return failedFuture(new IllegalArgumentException("Default distribution zone cannot be renamed."));
```

Comment:
Similar to aforementioned, I'd rather use something similar to "It's not possible to rename zone  to [name= " + DEFAULT_ZONE_NAME + ']'

Reply:
fixed